Godeps/_workspace/src/k8s.io/kubernetes/pkg/registry/generic/etcd/etcd.go

```diff
@@ -459,6 +457,11 @@ func (e *Etcd) DeleteCollection(ctx api.Context, options *api.DeleteOptions, lis
 			return nil, err
 		}
 	}
+
+	// Try to delete the directory, which will only succeed if the directory is empty.
+	// We can ignore errors since the cost of an empty directory is relatively small.
+	e.Storage.DeleteDir(ctx, e.KeyRootFunc(ctx));
```

Comment:
Yes, that is correct. We only want to delete namespaced resources. The comment was probably a typo.